client/src/components/LoadingError.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import CircleWarning from 'components/Icons/CircleWarning';
+
+/**
+ * Renders an error screen, used for when a schema fetch fails in the
+ * MFA app, or if a method is unavailable to be used but is already
+ * selected in the verification process.
+ *
+ * @param {string} title
+ * @param {string} message
+ * @param {mixed} controls
```

Comment:
Technically the (1) param is actually an object with those keys.

Reply:
Yeah I guess they're basically props. Can you think of a better way to document this?

Reply:
I read the docs. brb